config_ipa.go

```diff
@@ -27,7 +27,7 @@ package verkle
 
 import (
```

Comment:
I'd recommend to see this diff in "split" view since the default view can be confusing.